zerver/tests/test_subs.py

```diff
@@ -1038,14 +1038,14 @@ def test_change_stream_message_retention_days(self) -> None:
         user_profile = self.example_user('desdemona')
         self.login_user(user_profile)
         realm = user_profile.realm
-        do_change_plan_type(realm, Realm.LIMITED)
+        do_change_plan_type(realm, Realm.LIMITED, acting_user=user_profile)
```

Comment:
It's not necessary to pass `acting_user` in these tests since we are using `do_change_plan_type` only to change the plan type. We are not really concerned here who made the change since it's not something that we test or care about. So you can ignore the `acting_user` here as well as in other similar scenarios. Also make similar changes in other commits as well as applicable.